### PR TITLE
Add support for alt doc types

### PR DESCRIPTION
**Pull request by shanear**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,27 +3,10 @@ cache: bundler
...
-    - rvm: 2.2.3
-      before_install: gem install bundler --no-ri --no-rdoc
-      jdk: oraclejdk8
-    - rvm: jruby-1.7.22
-      before_install: gem install bundler --no-ri --no-rdoc
-      before_script: export JRUBY_OPTS="$JRUBY_OPTS --2.0"
-      jdk: oraclejdk7
-    - rvm: jruby-1.7.22
-      before_install: gem install bundler --no-ri --no-rdoc
-      before_script: export JRUBY_OPTS="$JRUBY_OPTS --2.0"
```
Comment (paultag) [inline .travis.yml]: Were these failing?

Comment (paultag) [inline .travis.yml]: (It's got a `--2.0`)

File: src/vbms/responses/document.rb [hunk 2]
```diff
@@ -1,28 +1,40 @@
...
             source: el['source'],
             mime_type: el['mimeType'],
             received_at: received_date.nil? ? nil : Time.parse(received_date.content).to_date)
       end
 
+      def self.extract_alt_doc_types(el)
+        if el['metadata']
+          metadata = JSON.parse(el['metadata'])
+
+          return JSON.parse(metadata['altDocType']) if metadata['altDocType']
```
Comment (paultag) [inline src/vbms/responses/document.rb]: :scream:

Comment (shanear) [inline src/vbms/responses/document.rb]: #jsonindepth

File: .travis.yml (hunk 1 above)
Comment (paultag) [inline .travis.yml]: We should re-enable if it was a real error; we should (in theory) support old jruby with `--2.0`

Comment (shanear) [inline .travis.yml]: @paultag it's not that they were failing, they were just taking a long time, and currently irrelevant.

**Pull request merged**